keps/sig-node/2033-kubelet-in-userns-aka-rootless/README.md

```diff
@@ -654,67 +798,200 @@ Note that this feature gate does not support separating kubelet's user namespace
 node components such as CRI.
 All the node components must run in the same user namespace.
 
-* **Does enabling the feature change any default behavior?**
+###### Does enabling the feature change any default behavior?
 
+<!--
+Any change of default behavior may be surprising to users or break existing
+automations, so be extremely careful here.
+-->
+
+The limitation is same as Rootless Docker, Podman, etc.
+See <https://rootlesscontaine.rs/caveats/>.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+<!--
+Describe the consequences on existing workloads (e.g., if this is a runtime
+feature, can it break the existing applications?).
 
-During Alpha, we will document what workloads will work and what will not work.
+Feature gates are typically disabled by setting the flag to `false` and
+restarting the component. No other changes should be necessary to disable the
+feature.
 
-* **Can the feature be disabled once it has been enabled (i.e. can we roll back
-  the enablement)?**
+NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
+-->
 
 N/A, as switching back rootless to rootful requires redeploying the kubelet, and vice versa.
```

Comment:
The correct answer here is - yes, it's possible to disable this functionality, by turning off the feature gate.

Reply:
Answered as yes